quetz/tasks/mirror.py

```diff
@@ -272,6 +274,41 @@ def initial_sync_mirror(
             version_stack.enter_context(method) for method in version_methods
         ]
 
+        update_batch = []
+        update_size = 0
+
+        def handle_batch(update_batch):
+            logger.debug(update_batch)
+            if not update_batch:
+                return False
+
+            remote_packages = []
+            try:
+                remote_packages.append(remote_repository.open(path))
```

Comment:
where does `path` come from? I thought you might iterate over paths in update_batch, but I can't see the iteration

Reply:
phew!